sklearn/tests/test_pipeline.py

```diff
@@ -165,6 +166,35 @@ def test_pipeline_fit_params():
     assert_true(pipe.named_steps['transf'].b is None)
 
 
+def test_pipeline_raise_set_params_error():
+
+    svm  = Pipeline([('kbest', SelectKBest()),
+                     ('cls', SVC())])
+
+    pipe = Pipeline([('pca', PCA()),
```

Comment:
The test doesn't touch them, does it? Just the `ova_svm` and `ova_svm__estimator__cls`. Or am I missing something?